types/lodash/index.d.ts

```diff
@@ -1344,7 +1345,7 @@ declare namespace _ {
          * // => { 'fred': 30, 'barney': 40 }
          */
         fromPairs<T>(
-          array: List<[StringRepresentable, T]> | null | undefined
+          array?: List<[PropertyName, T]> | null | undefined
```

Comment:
As per the same documentation, it also can't be `null` nor `undefined`. Shall we remove them?

Reply:
I created an issue a while ago (#15883) to add the `null | undefined` options to most functions that do non-mutating array operations. While it's not explicitly documented, lodash does handle `null` and `undefined` values gracefully. The reason being is that this saves you from needing to do a null check before calling these functions.

For example, if you have something like this:

```
function process(pairs?: List<[string, number]>) {
    const obj = _.fromPairs(pairs);
    ...
}
```

then you don't need to check if `pairs` is undefined - you can just use it. Lodash doesn't require people to do that check, so we shouldn't either. But I think we should at least require people to pass in an argument. `_.fromPairs()` is almost certainly a mistake, since it will always return `{}`, So it would be better for the programmer to just write `{}` instead of `_.fromPairs()`.